app/styles/_core.less

```diff
@@ -466,41 +466,77 @@ label.checkbox {
 
 .osc-secrets-form {
   .basic-secrets {
```

Comment:
Agreed.  I debated making that suggestion in my review, but figured it'd previously been reviewed and okayed as is.  Since @benjaminapetersen suggested it, mind moving the secrets LESS to a new partial, @jhadvig?